packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -465,6 +467,12 @@ class Uploader {
    * @param {any} stream
    */
   async _uploadTus (stream) {
+    testIdx++
+    if (testIdx > 1) {
+      this.failed = true
+      setTimeout(() => this.emitError(new Error('test')), 1000)
+      return Promise.reject(new Error('test'))
+    }
```

Comment:
Is this a deliberate change or was meant to be removed after testing?

Reply:
Ooh good catch this was taking along from a different branch.